AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -315,6 +315,7 @@ - (void)_initializeInstance
   _environmentState = ASEnvironmentStateMakeDefault();
   
   _calculatedDisplayNodeLayout = std::make_shared<ASDisplayNodeLayout>();
```

Comment:
We could but this would clutter our code base in a pretty bad way as every time we have to check if the `_calculatedLayout != nullptr` before accessing it. That's the reason I went with the safer and more readable approach imho.